src/rpm_ostree/cli_status.rs

```diff
@@ -1,18 +1,24 @@
 //! Interface to `rpm-ostree status --json`.
 
+use super::actor::{RpmOstreeClient, StatusCache};
 use super::Release;
 use failure::{bail, ensure, format_err, Fallible, ResultExt};
+use filetime::FileTime;
+use log::trace;
 use serde::Deserialize;
 use std::collections::BTreeSet;
+use std::fs;
+
+const OSTREE_DEPLS_PATH: &str = "/ostree/deploy";
```

Comment:
A docstring would be nice here, noting why we bypass rpm-ostree and directly go to ostree internals.